main.go

```diff
@@ -142,6 +156,8 @@ func main() {
 			RunScraper(conf)
 		} else if *mode == "webapp" {
 			RunServer(conf)
+		} else if *mode == "filesize_fetcher" {
```

Comment:
nitpick: should be `metainfo_fetcher` instead as it fetches more than just the file sizes